### PR TITLE
Make some datadirs.* config options more consistent 

### PR DESCRIPTION
**Pull request by beraldoleal**
This is a continuation from what I have started on #4553 and is part of #4443. Essentially is deprecating a few methods on `core/data_dir.py` since we have a new settings logic for a while now.

To avoid inconsistency code should get data from the settings dictionary. For now this is touching on `logs_dirs` and `data_dir`.

Signed-off-by: Beraldo Leal <bleal@redhat.com>

### CODE REVIEW

**Comment by beraldoleal** [comment]
The lint error here, it was already fixed by @ana, here: ddaec5e2b3ccade33f58ecc4c6bdf07865c94c52.

File: avocado/plugins/config.py [hunk 1]
```diff
@@ -100,8 +100,8 @@ def handle_default():
             LOG_UI.debug("file to customize values")
             LOG_UI.debug('')
             LOG_UI.info('Avocado Data Directories:')
-            LOG_UI.debug('    base     %s', data_dir.get_base_dir())
-            LOG_UI.debug('    tests    %s', data_dir.get_test_dir())
-            LOG_UI.debug('    data     %s', data_dir.get_data_dir())
-            LOG_UI.debug('    logs     %s', data_dir.get_logs_dir())
-            LOG_UI.debug('    cache    %s', ", ".join(cache_dirs))
```
Comment (ana) [inline avocado/plugins/config.py:107]: Out of curiosity: why did you remove one of the leading spaces here?

**Comment by ana** [comment]
Looks good to me, I have also tested it and works fine.
A small nitpick, commit 0059575e06f09683df0efc15573179d31589d2b3 mentions the command `avocado config --datadirs` with an extra s at the end.

File: avocado/plugins/config.py (hunk 1 above)
Comment (beraldoleal) [reply avocado/plugins/config.py:107]: Just to keep less then 80 columns.

**Comment by beraldoleal** [comment]
> Looks good to me, I have also tested it and works fine.
> A small nitpick, commit [0059575](https://github.com/avocado-framework/avocado/commit/0059575e06f09683df0efc15573179d31589d2b3) mentions the command `avocado config --datadirs` with an extra s at the end.

Nice catch. Thank you. Fixed with a force-push.

**Review by ana — APPROVED** [review]

**Review by willianrampazzo — APPROVED** [review]

**Review by clebergnu — APPROVED** [review]
LGTM, thanks!
